docs_source_files/content/_index.fr.md

```diff
@@ -59,13 +58,13 @@ from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support.expected_conditions import presence_of_element_located
 
-#Cet exemple requiert Selenium WebDriver 3.13 ou plus récent
+#Cet exemple requiert Selenium WebDriver 3.13 ou plus rÃ©cent
```

Comment:
Seems that this characted got messed up, if we can get it back the way it was we can then merge it. Thanks!

Reply:
@diemol . Reverted the text to original